doc/changelog.rst

```diff
@@ -1,6 +1,12 @@
 Changelog
 =========
 
+Version 4.1.0
+~~~~~~~~~~~~~
+
+**New feature**: Django 4.2.x, 5.x support.
+    Django EL(Endless) Pagination now supports Django 3.2.x, 4.2.x and 5.0
+    supports Python 3.8, 3.9, 3.10, 3.11, 3.12 (only with Django 5.0) 
```

Comment:
I do not think that Dj5 supports Py38 and Py39.
* https://docs.djangoproject.com/en/5.0/faq/install/#what-python-version-can-i-use-with-django